SharePointFramework/ProjectExtensions/src/components/DocumentTemplateDialog/TargetFolderScreen/index.tsx

```diff
@@ -54,9 +54,9 @@ export const TargetFolderScreen = () => {
         setFolder={setFolder}
       />
       <DetailsList
-        items={folders}
+        items={folders.sort((a, b) => (a.name > b.name ? 1 : -1)) }
         columns={columns()}
-        selectionMode={SelectionMode.multiple}
+        selectionMode={SelectionMode.none}
```

Comment:
We don't need selection here?

Reply:
At least as far as I can see, multiple selection makes noe sense here as it doesn't work selecting multiple folders for targets. If you know why multiple selection was set, please explain :)